test/commands/ledger/download.test.ts

```diff
@@ -1,84 +1,110 @@
 import {expect, test} from "@oclif/test"
 import * as fs from "fs"
+import { pipeline } from 'stream/promises';
+import { buildResponse, expectRequest } from '../../helpers/http';
 
 const tempDir = "./tmp/test"
 const ledgerFile = `${tempDir}/test-download.ndjson`
 const testToken = "test-token"
 const firstEventId = "0005d5813bf9c4498e2a8b74e95bff80"
 
+import { setMockCallback, initClient } from '../../../src/lib/client';
 
-describe("ledger:download", () => {
-  if (fs.existsSync(ledgerFile)) {
-    fs.rmSync(ledgerFile)
-  } else {
-    if (!fs.existsSync(tempDir)) {
-      fs.mkdirSync(tempDir, {recursive: true})
-    }
-  }
+describe("ledger:download", async () => {
+
+  it('should work', () => {
```

Comment:
Does this setup need to be in an `it()` ?

Reply:
Yes, because everything in describe gets executed at the start of the test suite, to collect the individual test cases.

This makes it very hard to debug individual tests. The real test cases should always be in the it() callbacks.

These days this is also built into node btw so we technically don't need mocha

Reply:
Do you think we should convert these to node's test framework? I have never used it so I don't know. I'll create a ticket for you to look into it.